index.js

```diff
@@ -49,4 +49,21 @@ module.exports = {
       },
     ],
   },
+  overrides: [
+    {
+      files: ["**/*.stories.(mdx|tsx)"],
+      extends: ["plugin:storybook/recommended"],
```

Comment:
Makes sense, and means we get the benefits of:
https://storybook.js.org/blog/how-to-use-storybook-with-eslint/